services/logs/logs.go

```diff
@@ -75,8 +72,7 @@ var defaultLogs = map[string]logInfo{
 
 	// core PMM components
 	"pmm-managed.log": {"/srv/logs/pmm-managed.log", "pmm-managed"},
-	"qan-api.log":     {"/srv/logs/qan-api.log", ""},
-	"qan-api2.log":    {"/var/log/qan-api2.log", ""},
+	"qan-api2.log":    {"/srv/logs/qan-api2.log", ""},
```

Comment:
Second parameter should not be empty. Please find systemd unit name for qan-api2 and use it.

Reply:
https://github.com/percona/pmm-server-packaging/blob/PMM-2.0/rhel/SPECS/percona-qan-api2.spec I guess it is `percona-qan-api2`.

@EvgeniyPatlan @hors Why we use `percona-qan-api2`, but `pmm-managed` there: https://github.com/percona/pmm-server-packaging/blob/PMM-2.0/rhel/SPECS/pmm-managed.spec ? Could we be more consistent?